include/tvm/relay/transform.h

```diff
@@ -571,6 +571,14 @@ TVM_DLL Pass EtaExpand();
  */
 TVM_DLL Pass PrintIR();
 
+/*!
+ * \brief Partition a Relay program into regions that can be executed on
+ * different backends.
+ *
+ * \return The pass.
+ */
+TVM_DLL Pass PartitionGraph();
```

Comment:
We should move away from graph terminology we should start to emphasize that we have more than a data-flow graph, this has led people to avoid scoping, effects, etc